katharsis-client/src/main/java/io/katharsis/client/internal/RelationshipRepositoryStubImpl.java

```diff
@@ -70,7 +70,7 @@ public void removeRelations(T source, Iterable<J> targetIds, String fieldName) {
 
 	private Serializable getSourceId(T source) {
 		ResourceField idField = sourceResourceInformation.getIdField();
-		return (Serializable) PropertyUtils.getProperty(source, idField.getUnderlyingName());
+		return (Serializable) idField.getAccessor().getValue(source);
```

Comment:
ResourceField grows a bit further with https://github.com/katharsis-project/katharsis-framework/pull/383. Some structure with access seems ok to avoid having to many methods on ResourceField, but I don't mind either.